cpp/proto/cert_serializer.cc

```diff
@@ -74,15 +74,16 @@ SerializeResult SerializeV1CertSCTSignatureInput(uint64_t timestamp,
   if (res != SerializeResult::OK) {
     return res;
   }
-  TLSSerializer serializer;
-  serializer.WriteUint(ct::V1, Serializer::kVersionLengthInBytes);
-  serializer.WriteUint(ct::CERTIFICATE_TIMESTAMP,
-                       Serializer::kSignatureTypeLengthInBytes);
-  serializer.WriteUint(timestamp, Serializer::kTimestampLengthInBytes);
-  serializer.WriteUint(ct::X509_ENTRY, Serializer::kLogEntryTypeLengthInBytes);
-  serializer.WriteVarBytes(certificate, kMaxCertificateLength);
-  serializer.WriteVarBytes(extensions, Serializer::kMaxExtensionsLength);
-  result->assign(serializer.SerializedString());
```

Comment:
My earlier comments about clearing the output string applies generally everywhere that was following that pattern, of course.